app/modules/munkiinfo/views/munkiinfo_munkiprotocol_widget.php

```diff
@@ -24,12 +24,12 @@
 	}
 
 		var panel = $('#munkiinfo-munkiprotocol-widget div.panel-body'),
-			baseUrl = appUrl + '/module/munkiinfo/listing';
+			baseUrl = appUrl + '/show/listing/munkireport/munki';
 		panel.empty();
 
 		// Set statuses
-		panel.append(' <a href="'+baseUrl+'#protocol = http" class="btn btn-danger"><span class="bigger-150">'+data.http+'</span><br>'+i18n.t('munkiinfo.munkiprotocol.http')+'</a>');
-		panel.append(' <a href="'+baseUrl+'#protocol = https" class="btn btn-success"><span class="bigger-150">'+data.https+'</span><br>'+i18n.t('munkiinfo.munkiprotocol.https')+'</a>');
+		panel.append(' <a href="'+baseUrl+'#protocol = http" class="btn btn-danger"><span class="bigger-150">'+data.http+'</span><br>&nbsp;'+i18n.t('munkiinfo.munkiprotocol.http')+'&nbsp;</a>');
```

Comment:
The spaces help make the module look more even and wide. Otherwise it would appear too tall and skinny.